simple_qa_rnn/relation_prediction/hyperparameter_tuning.py

```diff
@@ -10,8 +10,8 @@
     dropout = uniform(0.5, 0.6)
     clip = uniform(0.6, 0.7)
 
-    command = "python train.py --cuda --device 1 --dev_every 500 --log_every 250 --batch_size 128 " \
-                "--epochs {} --lr {} --d_hidden {} --n_layers {} --dropout_prob {} --clip {} >> " \
+    command = "python train.py --cuda --gpu 1 --dev_every 500 --log_every 250 --batch_size 32 " \
```

Comment:
We have 2 GPUs. I did this explicitly because there was this one time where someone (probably) you were using GPU index 0.